src/Orleans/Streams/PersistentStreams/StreamQueueBalancerType.cs

```diff
@@ -1,19 +1,22 @@
 namespace Orleans.Streams
 {
-    public enum StreamQueueBalancerType
+    /// <summary>
+    /// Built-in stream queue balancer type which is supported natively in orleans
+    /// </summary>
+    public static class BuiltInStreamQueueBalancerType
```

Comment:
I was concerned about breaking changes.  It seems like we could maintain the old enums and add support for strings, but in retrospect, it's probably best to have only one supported way (strings) and eat the cost of a breaking change.

I withdraw my suggestion.